src/interfaces/JoinUsFormInterface.ts

```diff
@@ -0,0 +1,36 @@
+interface IProps { }
+
+interface IState {
+  sliitId?: string;
+  name?: string;
+  email?: string;
+  mobileNumber?: string;
+  academicYear?: string;
+  selfIntroduction?: string;
+  reason?: string;
+  linkedInProfile?: string;
+  githubProfile?: string;
+  blog?: string;
+  experiences?: string;
+  challengesAndProblems?: string;
+  timeline?: string;
+  skills?: string[];
+  formErrors: {
```

Comment:
No need to have `formErrors` object. Now we use **Yup** package to handle validations. Remove the `formErrors` object and test the feature again.